test/UnitTests/MSTest.Analyzers.UnitTests/PreferTestCleanupOverDisposeAnalyzerTests.cs

```diff
@@ -17,15 +17,30 @@ public async Task WhenTestClassHasDispose_Diagnostic()
             using Microsoft.VisualStudio.TestTools.UnitTesting;
 
             [TestClass]
-            public class MyTestClass : IDisposable
+            public class MyTestClass : IDisposable, IMyInterface
```

Comment:
does it work with `IAsyncDisposable`?

Reply:
yeah the other test is using `IAsyncDisposable`